aldify/src/requests.ts

```diff
@@ -0,0 +1,28 @@
+import {Song} from "@/models/song";
+
+export function fetchAllSongs() : Promise<Song[]> {
+    return fetchSongs("*");
+}
+
+export function fetchSongs(query: string): Promise<Song[]> {
```

Comment:
nit: let's file an issue to swap out the mock backends with the real ones.

Reply:
Sure, thanks for mentioning.

Reply:
Created issue #5.